tests/Api/Shop/OrdersTest.php

```diff
@@ -25,11 +28,616 @@ final class OrdersTest extends JsonApiTestCase
 
     protected function setUp(): void
     {
+        $this->setUpDefaultGetHeaders();
         $this->setUpOrderPlacer();
 
         parent::setUp();
     }
 
+    /** @test */
+    public function it_gets_orders(): void
```

Comment:
```suggestion
    public function it_gets_only_logged_in_user_orders_excluding_guest_and_other_users_orders(): void
```
A bit long, but explicitly descriptive.